### DISCUSSION TITLE
Installing and Running Project

### DISCUSSION THREAD

**Discussion by alexia-narro** [aelurum/AssetStudioMod#26] [Q&A]
Hi guys! I'm a bit of a newbie with this stuff and was wondering how I would go about installing everything. I have Visual Studio 2022 downloaded, along with the .NET files and the FBX SDK. How do I run the project from here? From my understanding, I have to build the project but it says that it "cannot be built" in Visual Studio.

**Comment by aelurum** [comment]
Hi
There should be no problem if all the necessary dependencies ([FBX SDK](https://www.autodesk.com/content/dam/autodesk/www/adn/fbx/2020-2-1/fbx202021_fbxsdk_vs2019_win.exe) (with [pdb](https://www.autodesk.com/content/dam/autodesk/www/adn/fbx/2020-2-1/fbx202021_fbxsdk_vs2019_pdbs.exe)), .NET SDK 8) are installed correctly.
Just open the `AssetStudio.sln` file
Wait a bit while Visual Studio makes some preparations in the background
And click Build->Build Solution (F6)
You can also right-click on the AssetStudioGUI project in the Solution Explorer window and click `Set as Startup Project` to be able to start the GUI from Visual Studio
